ompi/mca/fs/lustre/fs_lustre_component.c

```diff
@@ -93,6 +95,15 @@ lustre_register(void)
                                            MCA_BASE_VAR_TYPE_INT, NULL, 0, 0,
                                            OPAL_INFO_LVL_9,
                                            MCA_BASE_VAR_SCOPE_READONLY, &mca_fs_lustre_stripe_width);
+    mca_fs_lustre_lock_algorithm = 0;
+    (void) mca_base_component_var_register(&mca_fs_lustre_component.fsm_version,
```

Comment:
nit (feel free to ignore) This could be more user friendly if it was implemented with mca_base_var_enum_t, but requires more code.